web/client/epics/feedbackMask.js

```diff
@@ -221,20 +224,22 @@ export const feedbackMaskPromptLogin = (action$, store) => // TODO: separate log
         .filter((action) => {
             const pathname = pathnameSelector(store.getState());
             return action.error
-                && [403].concat([LOAD_PERMALINK_ERROR].includes(action.type) ? 404 : []).includes(action.error.status)
-                && pathname.indexOf("new") === -1 && !(pathname.match(/(dashboard)/) !== null && pathname.match(/(dashboard)\/[0-9]+/) === null); // new map geostory and dashboard has different handling (see redirectUnauthorizedUserOnNewLoadError, TODO: uniform different behaviour)
+                && [403].concat([CONTEXT_LOAD_ERROR, LOAD_PERMALINK_ERROR].includes(action.type) ? 404 : []).includes(action.error.status)
+                && pathname.indexOf("new") === -1 && !(pathname.match(/dashboard/) !== null && pathname.match(/dashboard\/[0-9]+/) === null); // new map geostory and dashboard has different handling (see redirectUnauthorizedUserOnNewLoadError, TODO: uniform different behaviour)
         })
         .filter(() => !isLoggedIn(store.getState()) && !isSharedStory(store.getState()))
         .exhaustMap(
-            () =>
-                Rx.Observable.of(loginRequired()) // prompt login
+            () => {
+                const loginPage = getLoginPage();
+                const homePage = getMiscSetting('homePath', '/');
+                return Rx.Observable.of(loginPage ? goToPage(loginPage) : loginRequired()) // prompt login panel or redirect to login page
```

Comment:
We should not add this logic here.
The correct workflow is :
 - `loginRequred` --> someone reacts to it ( custom login logic in case of geOrchestra, standard login in case of mapstore).

Reply:
Yes, I shall remove it as per [this](https://github.com/geosolutions-it/MapStore2/pull/9298#discussion_r1277400981)